modules/unsupported/mbtiles/src/test/java/org/geotools/mbtiles/MBTilesMetadataTest.java

```diff
@@ -0,0 +1,12 @@
+package org.geotools.mbtiles;
+
+import org.junit.Test;
+
+public class MBTilesMetadataTest {
```

Comment:
One note here - the MBTilesFileTest class has a testMBTilesMetaData, so the class name for this test is a little misleading/redundant - it might be better if you just added your new test as its own test case to the MBTilesFileTest instead of creating a new test class.